ivy/functional/ivy/data_type.py

```diff
@@ -1027,11 +1027,18 @@ def set_default_float_dtype(float_dtype: Union[ivy.Dtype, str]):
 
 
 def set_default_int_dtype(int_dtype: Union[ivy.Dtype, str]):
-    """Summary.
+    """Default the type to int and append the type to stack.
 
     Parameters
     ----------
     int_dtype
+        int data type
+
+    Examples
+    --------
+    >>> ivy.set_default_int_dtype(int_dtype)
```

Comment:
please use dtype as an example parameter, eg. ivy.int8, ivy.native_int16, etc. 
a complete list can be found here: https://github.com/unifyai/ivy/blob/d80b093cccdbf5baf35bc6749f939998c130a445/ivy/__init__.py#L239